website/blog/2021-02-16-64updates.md

```diff
@@ -0,0 +1,203 @@
+- **isTouchExplorationEnabled** will return a boolean indicating whether or now the user has a screen narrator or similar running. This feature needs more testing, but is available in an alpha state for you to use - *partial implementation*.
+
+## 🎇 Expanded documentation
+We've added a lot to our documentation too, both within the Native APIs and on other important developer topics like virtual machines and picking C or C++ to have your app created in.
+
+### 🔧🔨 Choice between C++ or C# compiled apps
+A big request for our documentation has been steps on how to set up an app for C++ or C# based development, and we're happy to announce that we've [put together all the info necessary](https://microsoft.github.io/react-native-windows/docs/next/native-code-language-choice) to get started with either.
+
+### 📡 Develop for Windows on anything
+If you have either a local or Azure virtual machine set up, you can develop Windows app on a non-Windows PC.
+
+Grab the VM of your choice from the [Windows 10 development environment](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/). Once you have that virtual environment up and running, set up that new VM [set up for React Native for Windows development](https://microsoft.github.io/react-native-windows/docs/next/rnw-dependencies) just as you would today and deploy your builds to it.
+
+Check out the [full documentation on the process here](https://microsoft.github.io/react-native-windows/docs/next/win10-vm).
+
+### 💻 Native Windows APIs
+We've even added a reference sheet so you can take a look at the native API's that are available under the [`Microsoft.ReactNative`](https://microsoft.github.io/react-native-windows/docs/next/Native-API-Reference) namespace, like:
+
+- [Enums](https://microsoft.github.io/react-native-windows/docs/next/Native-API-Reference#enums)
+- [Interfaces](https://microsoft.github.io/react-native-windows/docs/next/Native-API-Reference#interfaces)
+- [Structs](https://microsoft.github.io/react-native-windows/docs/next/Native-API-Reference#structs)
+- [Classes](https://microsoft.github.io/react-native-windows/docs/next/Native-API-Reference#classes)
+- [Delegates](https://microsoft.github.io/react-native-windows/docs/next/Native-API-Reference#delegates)
+
+Curious about how to work with native code using a React Native for Windows app? [We've got you covered for that too!](https://microsoft.github.io/react-native-windows/docs/next/native-code)
+
+## 🚀 TurboModules
+
+We've added support for C++ JSI TurboModules as they exist today in React Native core projects. You won't have to worry about writing any code differently from how you do today in React Native, but when you run your app on Windows, we will ensure it works over the ABI-safe boundaries. This means your React Native Windows TurboModules can be shipped in different DLLs and we take care of the binary compatibility between versions.

```

Comment:
The ABI safety does not ensure binary compatibility between versions.  It ensures binary compatibility across DLLs on the same version.  I'm also not sure how many people care / want to know about the ABI compatibility right now.  -- This will likely be a rich topic once we move to binary distribution, but I think we can leave it out for now.

Maybe something like:

We now have support for C++ JSI TurboModules using the same TurboModule interface that is available on Android and iOS. Since these APIs haven't been fully shipped or documented in React Native core yet, these APIs are subject to change.